### ISSUE TITLE
[Bug][A11Y][iOS] Wrong accessible name for button text+icon

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#387]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

18.7

### App version

0.6.0

### Describe the issue

The button text+icon is read like this:
<img width="145" height="79" alt="" src="https://github.com/user-attachments/assets/41c02387-1035-4620-9362-f34c0feb0dbb" />

<img width="217" height="47" alt="" src="https://github.com/user-attachments/assets/7c1cd4bb-4389-4b7a-b0b4-cb019b618b0e" />

### Expected Behavior

The icon is decorative here. The button with text+icon should be read with only the text.

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (alpha) (456)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 429
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (456)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 429
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by B3nz01d** [comment]
@nouha06 
* For iOS I need to perform 2 swipes to get into the item that will read the accessibility information. The should only be one.
* For Android it's ok

**Comment by pya35** [comment]
> [@nouha06](https://github.com/nouha06)
> 
>     * For iOS I need to perform 2 swipes to get into the item that will read the accessibility information. The should only be one.

same for me

**Comment by nouha06** [comment]
I fixed this issue, we currently have a deployment problem for iOS on TestFlight, so I’ll deliver this fix later.
added a blocking label until we resolve the issue of deployment.

**Comment by Tayebsed93** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 429
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 429
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)
